### ISSUE TITLE
Incorrectly detects source language, impossible to fix

### ISSUE THREAD

**Issue by yuchdev** [FilipePS/Traduzir-paginas-web#632]
The extension detects wrong language of the page (English, while the actual language is Italian), but it seems impossible to point the mistake and forcibly translate from Italian.

**Comment by FilipePS** [comment]
Send me the website link so I can check it out.

**Comment by yuchdev** [comment]
I can't send unfortunately, this is intranet Wiki page, 20% in English, 80% in Italian, detected as English.
The thing is, I can't manually change the source language, or at least this is completely unclear how to do it

**Comment by FilipePS** [comment]
Even if the extension mistakenly detects the language of a website, it does not prevent it from being translated.
Even if you manually click on Translate, the site is not translated? any part of it?

**Comment by yuchdev** [comment]
I already translated the source page into English (this is why I opened it in the first place).
Yes, It attempted to translate from English, which was about 20% of the page. The content in Italian was not affected.
Since the page that caused the problem does not exist anymore, you are free to close the ticket as UTR.
Just remember, there's a problem with multiple languages pages, the used should have a chance to choose from what exactly language he wants translate. 
Thank you for your time

**Comment by Luni2000** [comment]
I agree. It sometimes happens (and there will always be cases), that the language is recognised incorrectly. This means, that the page is translated incorrectly.
I would like to be able to tell the extension from which language it should translate from. As I have context I oftentimes already know which language this is in or at least have a pretty good gues and just try if it gets translated.
